test/ng/directive/selectSpec.js

```diff
@@ -1888,7 +1888,14 @@ describe('select', function() {
             optionElements = element.find('option');
             expect(optionElements.length).toEqual(1);
             expect(scope.obj.value).toEqual([]);
-            expect(element.val()).toBe(null);
+
+            // Cover both jQuery 3.x ([]) and 2.x (null) behavior.
+            var val = element.val();
+            if (val === null) {
+              val = [];
+            }
+            expect(val).toEqual([]);
```

Comment:
Nitpick: I would prefer `expect(element.val()).toBeOneOf(null, [])` for readability.